tools/generator/filters/python.py

```diff
@@ -11,7 +11,7 @@
 
 @register_filter
 @pass_context
-def python_type(ctx, val: str) -> str:
+def python_type(ctx, val: str, tuple_alias: bool = True) -> str:
```

Comment:
Could you add a short documentation to the function? It's hard to understand what tuple_alias means here